engine/impl/Playlist.php

```diff
@@ -440,40 +454,56 @@ public function isDateTimeWithinShow($timeStamp, $listRow) {
 
     // insert playlist track. return following: 0 - fail, 1 - success no 
     // timestamp, 2 - sucess with timestamp.
-    public function insertTrack($playlistId, $tag, $artist, $track, $album, $label, $wantTimestamp, &$id = null) {
+    public function insertTrack($playlistId, $tag, $artist, $track, $album, $label,  $insertTime, &$id, &$status) {
         $row = $this->getPlaylist($playlistId, 1);
 
-        // log time iff 'now' is within playlist start/end time.
-        $doTimestamp = $wantTimestamp && $this->isNowWithinShow($row);
-        $timeName    = $doTimestamp ? "created, " : "";
-        $timeValue   = $doTimestamp ? "NOW(), "   : "";
+        $isWithinShow = $this->isWithinShow($insertTime, $row);
+        if (!$isWithinShow) {
+            $status = "Spin time is outside of show start/end times. ";
+            error_log($status);
```

Comment:
I'm not sure error_logging these is appropriate for production.  Fine to return a message to the caller, I don't think it is an operational error that we would expect to appear in the server logs.

Reply:
the flip side is that it is unlikely to add much volume to the logs and that having it in them helps when debugging issues involving pilot error that DJs have reported as code errors. please comment whether you still think that this should be removed.

Reply:
I'm not fussed about the logging and will accept the PR either way.